### PR TITLE
add cards index.html

### PR DESCRIPTION
**Pull request by boikoua**
❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:

- [DEMO LINK](https://boikoua.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://boikoua.github.io/layout_product-cards/report/html_report/)

❗️ Copy this `Checklist` to the `Pull Request` description after links, and put `- [x]` before each point after you checked it.

- [x] there 2 BEM blocks `card` and `stars` each in it own file
- [x] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x] SCSS Variables are used for main values and placed in a **separate file**
- [x] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,33 @@
...
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
```
Comment (anastasiiavorobiova) [inline src/index.html:14]:
```suggestion
    <article class="card" data-qa="card">
```
Consider leveraging semantic tags

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,33 @@
...
+      <div class="card__image"></div>
+      <div class="card__title">
```
Comment (anastasiiavorobiova) [inline src/index.html:16]:
```suggestion
      <h3 class="card__title">
```

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,33 @@
...
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">Product code: 195434</div>
```
Comment (anastasiiavorobiova) [inline src/index.html:19]:
```suggestion
      <p class="card__code">Product code: 195434</p>
```

File: src/index.html [hunk 4]
```diff
@@ -5,9 +5,33 @@
...
+      <div class="card__rate">
+        <div class="card__stars stars">
```
Comment (anastasiiavorobiova) [inline src/index.html:21]:
```suggestion
        <div class="card__stars stars stars--4">
```

File: src/index.html [hunk 5]
```diff
@@ -5,9 +5,33 @@
...
+          <div class="stars__star stars__star--1"></div>
+          <div class="stars__star stars__star--2"></div>
+          <div class="stars__star stars__star--3"></div>
+          <div class="stars__star stars__star--4"></div>
+          <div class="stars__star stars__star--5"></div>
```
Comment (anastasiiavorobiova) [inline src/index.html:26]:
```suggestion
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
```

File: src/index.html [hunk 6]
```diff
@@ -5,9 +5,33 @@
...
+        </div>
+        <div class="card__review">Reviews: 5</div>
```
Comment (anastasiiavorobiova) [inline src/index.html:28]:
```suggestion
        <p class="card__review">Reviews: 5</p>
```

File: src/index.html [hunk 7]
```diff
@@ -5,9 +5,33 @@
...
+      </div>
+      <div class="card__price">
```
Comment (anastasiiavorobiova) [inline src/index.html:30]:
```suggestion
      <p class="card__price">
```

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+  &__btn {
+    width: 100%;
+    border: none;
+    border-radius: 5px;
+    background: $btn-color;
+    padding-block: 12px;
+    margin-bottom: 16px;
+
+    color: #FFF;
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:86]: Consider using variables for repeating colors

File: src/styles/blocks/card.scss [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+    font-family: Roboto, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      border-radius: 5px;
+      border: 1px solid $btn-color;
+      background: #FFF;
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:97]: Consider using variables for repeating colors

File: src/styles/blocks/stars.scss [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+    background-image: url(./../images/star.svg);
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 7px;
+  }
+
+  &__star:nth-child(-n+4) {
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/stars.scss:16]: Consider using CSS from the previous task to keep modifiers working

File: src/styles/index.scss [hunk 11]
```diff
@@ -1,3 +1,11 @@
...
+@import './blocks/card';
+@import './blocks/stars';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:11]: Consider moving general styling to a separate file to keep your main file clean and clear

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added a few comments to fix

File: src/index.html (hunk 4 above)
Comment (anastasiiavorobiova) [reply src/index.html:21]: Not fixed

File: src/index.html (hunk 6 above)
Comment (anastasiiavorobiova) [reply src/index.html:28]: Not fixed

File: src/styles/blocks/stars.scss (hunk 10 above)
Comment (anastasiiavorobiova) [reply src/styles/blocks/stars.scss:16]: Not fixed

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great progress here! Almost done!

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Plz check previous comment before approve

**Comment by boikoua** [comment]
> Plz check previous comment before approve

I don't know what's wrong. I fixed it.

File: src/index.html (hunk 5 above)
Comment (maxim2310) [reply src/index.html:26]: Not fixed

File: src/index.html (hunk 4 above)
Comment (maxim2310) [reply src/index.html:21]: Not fixed

**Review by maxim2310 — CHANGES_REQUESTED** [review]
these comments not fixed

File: src/index.html (hunk 4 above)
Comment (etojeDenys) [reply src/index.html:21]: image

**Review by etojeDenys — CHANGES_REQUESTED** [review]
fix all prev comments

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
still not fixed from the previous review

**Pull request closed without merging**